src/core/redux/selectors/selectOrgDataSets.js

```diff
@@ -18,8 +25,13 @@ export default function selectOrgDataSets(organizationId :UUID, dataSetIds ?:UUI
       if (dataSetIds) {
         return Map().withMutations((mutableMap :Map<UUID, Map>) => {
           dataSetIds.forEach((dataSetId :UUID) => {
-            const dataSet = getIn(state, [EDM, ORG_DATA_SETS, organizationId, dataSetId]) || Map();
-            mutableMap.set(dataSetId, dataSet);
+            const entitySetIndex :number = state.getIn([EDM, ENTITY_SETS_INDEX_MAP, dataSetId], -1);
+            if (entitySetIndex >= 0) {
+              const dataSet :?EntitySet = state.getIn([EDM, ENTITY_SETS, entitySetIndex]);
+              if (dataSet && dataSet.id) {
+                mutableMap.set(dataSetId, dataSet.toImmutable());
+              }
+            }
```

Comment:
Please revert this change. The app has switched to the "data set" model and I believe this is no longer correct. In fact, I think we need to completely remove getting/storing entity sets (probably property types too), i.e. `[EDM, ENTITY_SETS]` and `[EDM, ENTITY_SETS_INDEX_MAP]`.

Reply:
so, to be clear, the getEntitySet and getEntitySets reducers should be refactored to map to ORG_DATA_SETS rather than ENTITY_SETS?

Reply:
so just to be clear, `getEntitySets` and `getEntitySet` should be mapped to `ORG_DATA_SETS` now?

Reply:
No. `getEntitySet` and `getEntitySets` are ok as they are, it's just that this `selectOrgDataSets` needs to be reverted to what it was before.